Source/WebCore/dom/IdleCallbackController.cpp

```diff
@@ -93,7 +87,9 @@ void IdleCallbackController::startIdlePeriod()
         m_runnableIdleCallbacks.append({ request.identifier, WTFMove(request.callback) });
```

Comment:
For deadline, right now, we are computing it per `IdleCallbackController`, but this means that if we have multiple documents, each document can run 50ms of tasks. Should we have a global deadline instead? (or per window?, I think the current spec text is per-window (though getDeadline implementation allows it to be more global one)).

Reply:
Right, I have a WIP PR to do that as a follow up.

Reply:
New PR up at https://github.com/WebKit/WebKit/pull/16426